include/cute/atom/mma_atom.hpp

```diff
@@ -928,6 +928,179 @@ print_latex_mma(LayoutC const& C, ThrIDC const& TC,  // (m,n) -> (tid,vid)  and
   printf(latex_footer);
 }
 
+// MNK MMA Layout to SVG -- 8-value color coded by thread
+template <class LayoutC, class ThrIDC,
+          class LayoutA, class ThrIDA,
+          class LayoutB, class ThrIDB>
+CUTE_HOST_DEVICE
+void
+print_svg_mma(LayoutC const& C, ThrIDC const& TC,  // (m,n) -> (tid,vid)  and  tid -> thr_idx
+                LayoutA const& A, ThrIDA const& TA,  // (m,k) -> (tid,vid)  and  tid -> thr_idx
+                LayoutB const& B, ThrIDB const& TB)  // (n,k) -> (tid,vid)  and  tid -> thr_idx
```

Comment:
Textual alignment.
This should also parameterize the coloring like all of the other print_latex interfaces.

from @ccecka

Reply:
These were comments on the internal MR. The color parameterization isn't released publically yet, @hwu36, so @reed-lau won't be able to follow the same interface.

Reply:
@hwu36  I correct the indentation error to align with the formatting of the other code.